shaders/shader_path.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <string>
+
+inline std::string rgen_shader_path = "${rgen_shader_path}";
```

Comment:
Are these environment variables? Where are they set?

Reply:
These variables are set by CMakeLists.txt. That will be replaced by shader's absolute path.